lib/Capabilities.php

```diff
@@ -145,6 +145,10 @@ public function getCapabilities(): array {
 			$capabilities['features'][] = 'reactions';
 		}
 
+		if ($this->talkConfig->isBreakoutRoomsEnabled()) {
+			$capabilities['features'][] = 'recording-v1';
```

Comment:
1. Wrong config function
2. I would split this into 2 things (like with breakout rooms):
    1. A `recording-v1` which signals the API level
    https://github.com/nextcloud/spreed/blob/f121cccdab849dc5a5d2a960e8665f8adbb6d388/lib/Capabilities.php#L115
    3. A config => call => recording which signals the enabled/disabled:
    https://github.com/nextcloud/spreed/blob/f121cccdab849dc5a5d2a960e8665f8adbb6d388/lib/Capabilities.php#L124